### ISSUE TITLE
Package won't build

### ISSUE THREAD

**Issue by asaldivar** [mubaidr/prerender-plugin#10]
Thanks for working on revamping the `prerender-spa-plugin`!

Receiving the following error when I *only* require `prerender-plugin` in my webpack file:

```
async function process (compilation, done) {
      ^^^^^^^^
SyntaxError: Unexpected token function
```

**Comment by mubaidr** [comment]
What node version? 

This plugin requires node >= 7.5

**Comment by asaldivar** [comment]
I was using node 6 but just installed 7.5 and it's still not working. I think I may not be installing version 7.5 correctly.

Are the following commands correct:
`nvm install 7.5`
`nvm use 7.5`

Then when I ran `node --version` I get back `v7.5.0` but when I then try running npm run build. I get the same error. Am I missing a command in updating my version of node?

**Comment by mubaidr** [comment]
I am sorry, required version is 7.6.0.

**Comment by asaldivar** [comment]
Thank you! That was it! I know have the package running with node version 7.6.

**Comment by mubaidr** [comment]
Great.

This is because of the async feature not available in nodejs versions less than 7.6

**Comment by asaldivar** [comment]
Makes total sense. Thanks again for your speedy helpful reply.
